src/components/PolicyMasterPanel.js

```diff
@@ -202,6 +202,11 @@ class PolicyMasterPanel extends FormPanel {
         ),
       });
     }
+    const minimumPolicyEffectiveDate = this.props.modulesManager.getConf(
+      "fe-policy",
+      "minimumPolicyEffectiveDate",
+      0
+    );
```

Comment:
I'm not sure if this is the right place to declare such a variable. I believe that the component is a class, so you should declare it in the constructor.

This declaration shouldn't cause any errors, but I would change it to be consistent with the code base.